packages/modeling/src/operations/modifiers/snap.d.ts

```diff
@@ -1,6 +1,6 @@
 import { Geometry } from '../../geometries/types'
 import RecursiveArray from '../../utils/recursiveArray'
 
-export function snap<T extends Geometry>(geometry: T): T
-export function snap<T extends Geometry>(...geometries: RecursiveArray<T>): Array<T>
-export function snap(...geometries: RecursiveArray<Geometry>): Array<Geometry>
+export default function snap<T extends Geometry>(geometry: T): T
+export default function snap<T extends Geometry>(...geometries: RecursiveArray<T>): Array<T>
+export default function snap(...geometries: RecursiveArray<Geometry>): Array<Geometry>
```

Comment:
I don't even know what happens if the default is defined multiple times. I suggest reverting.